dev/inventory

```diff
@@ -1,3 +1,2 @@
 [is]
-172.28.128.4
-172.28.128.5
+wso2is ansible_host=172.28.128.4
```

Comment:
Remove the static IP and add a comment in the Readme to add the relevant IP of the instance to the inventory